Cargo.toml

```diff
@@ -37,8 +37,8 @@ chrono = "0.4.2"
 byteorder = "^1.2"
 lazy_static = "1.1.0"
 x25519-dalek = { version = "0.3", default-features = false, features = ["std", "u64_backend"] }
-prost = { git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
-prost-derive = {git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
+prost-amino = { path = "../amino_rs/"}
+prost-derive = {path = "../amino_rs/prost-derive/"}
```

Comment:
Note to myself: this should be changed when https://github.com/tendermint/amino_rs/pull/14 is merged.